src/custom-element.js

```diff
@@ -707,6 +704,7 @@ function createBaseCustomElementClass(win) {
         this.implementation_.layoutWidth_ = this.layoutWidth_;
       }
       // TODO(malteubl): Forward for stubbed elements.
+      // TODO(jridgewell): We should pass the layoutBox down.
```

Comment:
> But we'd keep getLayoutBox() method itself?

Yup, we need that else where.

> Ideally this would be only size.

Yah, just passing `{ width, height }` would satisfy most all of them. Anyone else could call `#getPosition` or whatever we end up calling it.